examples/webgpu_skinning_instancing.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js - WebGPU - Skinning Instancing</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+		<!-- WebGPU (For Chrome 94-101), expires 09/01/2022 -->
+		<meta http-equiv="origin-trial" content="AoS1pSJwCV3KRe73TO0YgJkK9FZ/qhmvKeafztp0ofiE8uoGrnKzfxGVKKICvoBfL8dgE0zpkp2g/oEJNS0fDgkAAABeeyJvcmlnaW4iOiJodHRwczovL3RocmVlanMub3JnOjQ0MyIsImZlYXR1cmUiOiJXZWJHUFUiLCJleHBpcnkiOjE2NTI4MzE5OTksImlzU3ViZG9tYWluIjp0cnVlfQ==">
+	</head>
+	<body>
+
+		<div id="info">
+			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> - WebGPU - Skinning Instancing
+		</div>
+
+		<script async src="https://unpkg.com/es-module-shims@1.3.6/dist/es-module-shims.js"></script>
+
+		<script type="importmap">
+			{
+				"imports": {
+					"three": "../build/three.module.js",
+					"three-nodes/": "./jsm/nodes/"
+				}
+			}
+		</script>
+
+		<script type="module">
+
+			import * as THREE from 'three';
+			import * as Nodes from 'three-nodes/Nodes.js';
+
+			import { FBXLoader } from './jsm/loaders/FBXLoader.js';
+
+			import WebGPU from './jsm/capabilities/WebGPU.js';
+			import WebGPURenderer from './jsm/renderers/webgpu/WebGPURenderer.js';
+
+			let camera, scene, renderer;
+
+			let mixer, clock;
+
+			init().then( animate ).catch( error );
+
+			async function init() {
+
+				if ( WebGPU.isAvailable() === false ) {
+
+					document.body.appendChild( WebGPU.getErrorMessage() );
+
+					throw new Error( 'No WebGPU support' );
+
+				}
+
+				camera = new THREE.PerspectiveCamera( 50, window.innerWidth / window.innerHeight, 1, 1000 );
+				camera.position.set( 100, 200, 300 );
+
+				scene = new THREE.Scene();
+				camera.lookAt( 0, 100, 0 );
+
+				clock = new THREE.Clock();
+
+				//lights
+
+				const centerLight = new THREE.PointLight( 0xffffff, .8, 7000 );
+				centerLight.position.y = 450;
+				centerLight.position.z = - 200;
+				scene.add( centerLight );
+
+				const cameraLight = new THREE.PointLight( 0x0099ff, .7, 7000 );
+				camera.add( cameraLight );
+				scene.add( camera );
+
+				const loader = new FBXLoader();
+				loader.load( 'models/fbx/Samba Dancing.fbx', ( object ) => {
+
+					mixer = new THREE.AnimationMixer( object );
+
+					const action = mixer.clipAction( object.animations[ 0 ] );
+					action.play();
+
+					const instanceCount = 50;
+					const dummy = new THREE.Object3D();
+
+					object.traverse( ( child ) => {
+
+						if ( child.isMesh ) {
+
+							child.material = new Nodes.MeshStandardNodeMaterial();
+
+							child.isInstancedMesh = true;
```

Comment:
Well, this is more or less a workaround until we figure out a better way to combine skinning + instancing. For now, I think it's okay.

Reply:
> Well, this is more or less a workaround until we figure out a better way to combine skinning + instancing. For now, I think it's okay.

What do you think about an API like this?

```js
const instanceCount = 10;
const instancedObj3d = new InstanceObject3D( mesh || sprite3d || skinnedMesh, instanceCount );

scene.add( instancedObj3d  );
```

Reply:
> Well, this is more or less a workaround until we figure out a better way to combine skinning + instancing. For now, I think it's okay.

What do you think about an API like this?

```js
const instanceCount = 10;
const instancedObj3d = new InstanceObject3D( mesh || sprite3d || skinnedMesh, instanceCount );

scene.add( instancedObj3d  );
```